modules/genomic_browser/ajax/genomic_file_upload.php

```diff
@@ -0,0 +1,471 @@
+<?php
+/**
+ * This allow to upload file for the genomic module.
+ *
+ * NOTE : This require a genomic_uploader directory under the
+ * GenomicDataPath directory specified in the cconfigSettings.
+ * This directory needs to be accessible w/r to the apache user.
```

Comment:
This Note paragraph - fantastic.  It should (also?) go in the Readme. 

little things: cconfigSettings;  require->requires, spell out w/r (with respect)
accessible -> executable